src/components/navigators/BottomNavigator/BottomNavigator.style.ts

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+
+export const BottomNavContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  padding: 1rem;
+  background-color: ${(props: any) => props.theme.container_color};
+  align-items: center;
+  z-index: 1000;
+`;
+
+export const BottomNavItem = styled.div<{ active: boolean }>`
```

Comment:
[ P2 ] 보통 styled components 사용할때 prop이 있다면 prop의 이름 앞에 `$`를 붙여주더라구요!
실제로 지난번 프로젝트때 저도 안붙여줬을때 에러가 생기는 경우도 많았어서 붙이시면 좋을 것같습니다!

```suggestion
export const BottomNavItem = styled.div<{ $active: boolean }>`
```

Reply:
오! 스타일링을 위한 props를 구분하면 진짜 좋겠네요 ㅎㅎ 에러 방지를 위해서라도 꼭 붙이겠습니다~!!

Reply:
저도 통일 시키는 게 좋겠네요..! 👍